.github/.wordlist.txt

```diff
@@ -887,6 +888,8 @@ SDK's
 SDKs
 SDKTARGETSYSROOT
 sdl
+sed
+SED
```

Comment:
nit: I believe this file is case-insensitive so we may be able to only enter one of these.

Reply:
done